Models/Chat.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace Tinker_Back.Models;
-
-public partial class Chat
+namespace Tinker_Back.Models
 {
-    public int Id { get; set; }
 
-    public string Name { get; set; } = null!;
+    public class Chat
+    {
+        public Chat(string name, string description)
```

Comment:
Accordring to line 20, descroption can be nullable, but here, in constructor we need to pass only string, can it be changed?